src/__tests__/authenticate.test.ts

```diff
@@ -0,0 +1,33 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as apiClient from '../';
+import { valid_data, invalid_data } from './fixtures/authenticate';
+
+const mock = new MockAdapter(axios);
+
+afterEach(() => {
+	mock.reset();
+});
+
+const apiClientInit = new apiClient.APIClient({ apiBase: 'unics-social' });
+
+test('authenticate(): authenticates valid users correctly', async () => {
+	mock.onPost('unics-social/authenticate').reply(200, {
+		status: {
```

Comment:
So authenticate's response body is just an object containing token, so this should be:

```js
obj.reply(200, { token: 'tokenhere' });
```

This applies to the test below too.